include/swift/SIL/MemAccessUtils.h

```diff
@@ -1264,10 +1267,14 @@ SILBasicBlock::iterator removeBeginAccess(BeginAccessInst *beginAccess);
 
 namespace swift {
 
-/// Return true if \p svi is a cast that preserves the identity and
-/// reference-counting equivalence of the reference at operand zero.
+/// Return true if \p svi is a cast that preserves the identity equivalence of
+/// the reference at operand zero.
 bool isRCIdentityPreservingCast(SingleValueInstruction *svi);
 
+/// Return true if \p svi is a cast that preserves the identity equivalence and
+/// reference-counting equivalence of the reference at operand zero.
+bool isRCIdentityOwnershipPreservingCast(SingleValueInstruction *svi);
```

Comment:
I don't really understand what these names mean any more. Maybe

`isIdentityPreservingRefCast`
and
`isOwnershipPreservingRefCast`
?